--- /etc/init.d/logstash.orig	2016-05-19 17:54:36.004217940 +0000
+++ /etc/init.d/logstash	2016-05-19 17:54:47.164340285 +0000
@@ -97,7 +97,7 @@
       sleep 1
     done
     if status ; then
-      if [ "$KILL_ON_STOP_TIMEOUT" == 1 ] ; then
+      if [ "$KILL_ON_STOP_TIMEOUT" -eq 1 ] ; then
         echo "Timeout reached. Killing $name (pid $pid) with SIGKILL. This may result in data loss."
         kill -KILL $pid
         echo "$name killed with SIGKILL."
